README.md

```diff
@@ -68,4 +68,18 @@ Guy Itzhaki, Tarel Madar, Gal Lapid, Alon Weissfeld, Omri Rosner
 1. Install VirtualBox and Vagrant.
 2. Clone this repository.
 3. Open a shell prompt, and switch to the location of the repository.
-4. Run 'vagrant up'. Once it finished, you're all set!
\ No newline at end of file
+4. Run 'vagrant up'. Once it finished, you're all set!
+
```

Comment:
The instructions below should not be needed, `vagrant up` and `vgrant halt` should be the only command that other developers need - unless thry need to connect to the DB directly or something like that.

If people need to run commands like below, you are not meeting the 4th milestone requirements.